### ISSUE TITLE
cannot delete some of the pregenerated entries (using mysql)

### ISSUE THREAD

**Issue by 0spinboson** [bolt/core#784]
Just created a new project, bolt 4 beta 5, php 7.2, and am trying to delete some of the auto-generated content that came with the fresh installation. About half of the entries can be deleted fine, the other half consistently gives these kinds of errors:
[error.txt](https://github.com/bolt/core/files/3946188/error.txt)

**Comment by bobdenotter** [comment]
This bit: 

```
Doctrine\DBAL\Exception\ForeignKeyConstraintViolationException:
An exception occurred while executing 'DELETE FROM bolt_content WHERE id = ?' with params [7]:

SQLSTATE[23000]: Integrity constraint violation: 1451 Cannot delete or update a parent row: a foreign key constraint fails (`u8184p5640_dzgd`.`bolt_relation`, CONSTRAINT `FK_3431ED74A3934190` FOREIGN KEY (`to_content_id`) REFERENCES `bolt_content` (`id`))
```

Would indicate that there's an error with deleting records that have a relation to another table (hence the "Foreign key"). It doesn't break on SQlite, but I see it _does_ break on MySQL for me.

**Comment by bobdenotter** [comment]
Fix in the pipeline, here: #797
